setup.py

```diff
@@ -32,7 +32,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-VERSION = '0.0.9'
+VERSION = '0.1.0'
```

Comment:
I've previously used bumpversion for this: https://github.com/ome/omero-signup/tree/v0.0.9#release-process

If we move to a fully manual process https://github.com/ome/omero-signup/blob/v0.0.9/.bumpversion.cfg should be deleted and the README update

Reply:
I'll revert and give it a try.